packages/components/src/test/tree/controlled/UsePresentationTreeState.test.ts

```diff
@@ -61,12 +62,16 @@ describe("usePresentationTreeState", () => {
     } as RulesetVariablesManager);
 
     presentationManager.getNodesAndCount.resolves({ count: 0, nodes: [] });
+    const localization = new EmptyLocalization();
 
+    sinon.stub(IModelApp, "initialized").get(() => true);
     sinon.stub(Presentation, "presentation").get(() => presentationManager);
+    sinon.stub(Presentation, "localization").get(() => localization);
     sinon.stub(IModelApp, "quantityFormatter").get(() => ({
       onActiveFormattingUnitSystemChanged,
     }));
-    await UiComponents.initialize(new EmptyLocalization());
+    await Presentation.initialize();
+    await UiComponents.initialize(localization);
```

Comment:
Can you replace these with `sinon.stub`?

Reply:
Replaced `Presentation.initialize()` with a stub.

Reply:
Okay, I have no idea what this was about. I added it yesterday because it was complaining `Presentation` was not initialized when calling `loadNode`, but now it seems to work fine for whatever reason 😃